src/datasources/locking-api/locking-api.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Global, Module } from '@nestjs/common';
+import { ILockingApi } from '@/domain/interfaces/locking-api.interface';
+import { LockingApi } from '@/datasources/locking-api/locking-api.service';
+
+@Global()
```

Comment:
The module doesn't need to be a global one.

Reply:
Removed in 53082b79be2c075827de2f2c059f984030fbe1b8.